src/ankh/ankh.go

```diff
@@ -40,19 +40,57 @@ type Context struct {
 // AnkhConfig defines the shape of the ~/.ankh/config file used for global
 // configuration options
 type AnkhConfig struct {
-	CurrentContextName string             `yaml:"current-context"` // note the intentionally offset names here
-	CurrentContext     Context            `yaml:"-"`               // (private) filled in by code
-	Contexts           map[string]Context `yaml:"contexts"`
+	SupportedEnvironments []string           `yaml:"supported_environments"`
+	SupportedProfiles     []string           `yaml:"supported_profiles"`
+	CurrentContextName    string             `yaml:"current_context"`
```

Comment:
I took the easy way out for the PR and switched everything to underscores. I figured since even kubernetes can't keep it straight (uses camel case in resource definitions, and snake case in kubectl configs) then we aren't hurting much to use underscores. I figure consistency is more important that perfect alignment with another tool.

Reply:
Seems like a good call to me

Reply:
keep this consistent with kubectl: "current-context"
then, mutual consistency with other configs means we have: "supported-environments" and "supported-resource-profiles"